mbuild/compound.py

```diff
@@ -2436,11 +2436,13 @@ def to_parmed(self, box=None, title='', residues=None, show_ports=False,
                 try:
                     atomic_number = AtomicNum[atom.name.capitalize()]
                 except KeyError:
-                    element = element_by_name(atom.name.capitalize())
                     if name not in guessed_elements:
                         warn(
-                            'Guessing that "{}" is element: "{}"'.format(
-                                atom, element))
+                            'Cannot infer mass of "{}" since its name does '
```

Comment:
I think adding an element attribute along with the mass attribute will be great. Fewer warnings for correctly working with non-atomistic systems will be welcome.